gbdxtools/simpleworkflows.py

```diff
@@ -378,10 +378,10 @@ def generate_task_workflow_json(self, output_multiplex_ports_to_exclude=None):
 
 
 class Workflow(object):
-    def __init__(self, tasks, **kwargs):
+    def __init__(self, tasks=[], workflow_id=None, **kwargs):
         self.workflow = WF()
         self.name = kwargs.get('name', str(uuid.uuid4())[:8] )
-        self.id = None
+        self.id = workflow_id
```

Comment:
We probably don't want to do this because it's ambiguous whether or not you can set the workflow ID to an arbitrary value.